spec/features/saml/aal3_required_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+describe 'AAL3 authentication required in an SAML context' do
+  include SamlAuthHelper
+
+  describe 'SAML requesting AAL3 authentication' do
```

Comment:
These tests will need to be written for reals once we've updated the SAML code to allow partners to send multiple auth contexts so they can request AAL3 on a per-request basis.

Reply:
maybe through a `pending` clause in there instead of `xcontext`? It's easier to find IMO:

```ruby
  describe 'SAML requesting AAL3 authentication' do
    pending 'actual support of multiple auth contexts in SAML'

    context 'user does not have aal3 auth configured' do
      # ...
    end
  end
```